English translation for appdev docs (#76)
personium/new-docs: src/getting-started/appdev-introduction.md

---
id: appdev-introduction
title: Develop an app
sidebar_label: Introduction
---

> In this chapter, we will explain how to develop an application using Personium API through the flow of sample application development.

Reviewer: ```suggestion
In this chapter, we will explain how to develop an app using Personium APIs through the flow of sample app development.
```

Reviewer: Not sure if we want to use chapter here. Normally we don't know this is actually a chapter or a section. Should use different expression.

Author: We will unify the term chapter or sector in whole documents later.

## Sample app development

> Sample application (Personium Trails) is to store the movement history data accumulated on Google in Personium and share a part of the data with others. The type of application is a web application that runs on a browser.

Reviewer: ```suggestion
This sample app (Personium Trails) is a web application that runs on a browser. It allows the user to store the location history data accumulated from Google onto Personium and share selected data with third parties.
```

> Below is a screenshot of the completed sample app.

Reviewer: ```suggestion
Below is a screenshot of the sample app.
```

Reviewer: "completed" has no meaning here.

![Movement history data list](assets/getting-started/trails_locations_public.png)

> The code of the completed sample app can be found at the link below.

Reviewer: ```suggestion
The source code of the sample app is available on GitHub ([app-personium-trails](https://github.com/personium/app-personium-trails)).
```

> [app-personium-trails](https://github.com/personium/app-personium-trails)

Reviewer: ```suggestion

```

## Prior knowledge

> It is assumed that you have the following prior knowledge.

Reviewer: ```suggestion
The following knowledges are required before you continue reading.
```

* Basic web application development
* Front-end development (Html, JavaScript, CSS)
* REST API

## Advance preparation

> In the sample application, 2 Cell creation and Account, Role, ACL settings related to Cell are done in advance. When developing an app that uses Personium, do the same thing. Refer to [Previous Chapter Tutorial](../unit-administrator/tutorial.md) for the setting method related to cell creation.

Reviewer: ```suggestion
In the sample app, the following operations are performed in advance.

* create two Cells  
* configure account, role and ACL settings in each Cell  

When developing an app that uses Personium, please perform the same operations. Refer to [Previous Chapter Tutorial](../unit-administrator/tutorial.md) for the APIs relevant to Cell creation.
```

> The usage and details of the two cells in the sample app are shown in the table below.

Reviewer: ```suggestion
The descriptions of the two cells used in the sample app are shown in the table below.
```

> |Use|Cell name|Account name|Role name linked to Account|Role permission settings|

Reviewer: ```suggestion
|Cell type|Cell name|Account name|Role name linked to Account|Role permission settings|
```

|----|---|---------|-----------------------|--------------|
|Data subject Cell|alice.example|me|admin|root|
|App Cell|app-personium-trails.example|app|admin|root|

> When developing your own application, the names of each Cell, Account, and Role do not have to be as shown in the table because they are arbitrary.

Reviewer: ```suggestion
When developing your own app, you do not have to use the same names as the sample's.
```
